### DISCUSSION TITLE
Elmcreek (Standard LS22 map) Strommasten werden nicht korrekt erkannt

### DISCUSSION THREAD

**Discussion by LSxNano** [Courseplay/Courseplay_FS22#1709] [General]
Genauer gesagt z.b. auf Feld 1 gibt es am Rand so strommasten die sieht/erkennt er oft net und fährt einfach dagegen XD 
fix für die zukunft wäre da echt geil XD

**Comment by LSxNano** [comment]
Geht dabei speziell um des Kartoffelerntefahrzeug Grimme Ventor 4150 (standard LS22 Fahrzeug) hab das feld mit 12 headlands geplant und er will trotzdem ausm feld rausfahren zum wenden :(

**Comment by LSxNano** [comment]
![image](https://user-images.githubusercontent.com/107652305/174183915-12bc71f2-8fb5-4527-87a2-232a09139898.png)

**Reply by LSxNano** [reply]
![image](https://user-images.githubusercontent.com/107652305/174184484-ea33ca28-d553-4de3-bc87-6d9e952126b8.png)

**Reply by LSxNano** [reply]
![image](https://user-images.githubusercontent.com/107652305/174184860-cbbbee8a-9dd8-42e9-b7ac-105451b7146b.png)

**Comment by Tensuko** [comment]
Die Strommasten hat Giants zu dicht an den Feldrand gestellt, Teils in das Feld rein.
Im Kursgenerator gibt es ein Setting dafür um die Feldgrenze selber nach außen oder innen zu schieben.
Erklärung steht beim Setting und im ingame hilfe Menü.

**Reply by LSxNano** [reply]
und woran liegt es das der kartoffelernter nicht ausm feld raus fährt beim rückwärts fahren (fruchtumfahrung und wenden auf feld deaktiviert) weil er setzt halt immer einen wegpunkt zuspät wieder an nach der wende so das immer was stehen bleibt

![image](https://user-images.githubusercontent.com/107652305/174442187-a81c3fba-1d62-4808-a3f6-8aaab467ca56.png)

**Reply by LSxNano** [reply]
![image](https://user-images.githubusercontent.com/107652305/174443207-1e9321d1-5ee5-41f9-bf7a-3d5f3c95c0cc.png)
![image](https://user-images.githubusercontent.com/107652305/174445473-7c6edb05-cba8-4985-890e-9b831aa504c4.png)

**Reply by LSxNano** [reply]
ich weis ja net wie des genau programmiert wird geschweige denn habe ich ahnung davon XD aber für mich sieht des so aus als wäre der punkt vom fahrzeug wo der erntet zu weit hinten festgelegt im courseplay, falls man versteht was ich meine XD 
Sinn würde es machen wenn er direkt vorne an der kante des werkzeuges liegen würde falls er das net schon tut, da dort die ernte beginnt.
aussehen tut es als würde er unter der Fahrerkabine liegen.

**Reply by Tensuko** [reply]
Es gibt keinen "punkt wo er erntet in courseplay"...

Für das Gerät wird mehr vorgewende gebraucht, oder wenden auf dem Feld ausschalten oder Pathfinder Turn probieren (Fahrzeugeinstellung). Aber dafür wird auch das Feld viel zu klein sein......

**Reply by LSxNano** [reply]
vorgewenden fährt er zu früh an, egal ob die kurve auf scharf (klappt auch nur richtig wenn feld 4 eckig ist), weich oder rund steht
wenden auf dem feld ist aus da wir keine fruchtzerstörung haben kann er ruhig rausfahren
pathfinder turn ist aktiv genau wie das er pathfinder zum rückwärts fahren nutzen darf
beim automatisch abladen am hänger (schon zum anfang der ernte am feld platziert) kommt ihm das schneidwerk in die quere und er schiebt den anhänger weg (eventuell falsche berechnung der länge des fahrzeuges?)
